packages/dom/package.json

```diff
@@ -19,6 +19,7 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"react-native": "src/index",
```

Comment:
Since this effectively means we're starting to use the `@wordpress/dom` package, I wonder if you have some insight on what are we using it for? Like, I understand is for _pasting_, but I wonder what parts of the process are actually using it.

It's not easy to follow the code to see where exactly it's used so, having some general idea is still useful if you have it.

Reply:
When the `pasteHandler` function calls `deepFilterHTML`, it uses the `specialCommentConverter` filter. The `raw-handling/special-comment-converter.js` file has a [dependency on `@wordpress/dom`](https://github.com/WordPress/gutenberg/blob/master/packages/blocks/src/api/raw-handling/special-comment-converter.js#L4), and makes use of the `remove` and `replace` functions from there.

There is a comment in the `rawHandler` function describing the use of the `specialCommentConverter` filter there: `// Needed to create more and nextpage blocks.` As `pasteHandler` uses the same filter and calls through to the same function (`deepFilterHTML`) I believe this comment's description applies to the filter's use in `pasteHandler` as well.

By the way, deeper into this codepath, where `deepFilterNodeList` makes use of the filters (the forEach loop) is where the need arises for the `contains` method (which had to be patched into jsdom-jscore). In the forEach loop, `item` is actually the filter function.

FYI: If you had run the bundler previously (`yarn start`), and subsequently removed the line in question `"react-native": "src/index",` from `packages/dom/package.json`, you will not run into an error until you run `yarn start:reset`.

Reply:
Ah, nice! This is solid info so, let's add it to the PR description, cool?

> you will not run into an error until you run `yarn start:reset`

Right, makes sense. Metro caches the packages resolution anyway, so we need to reset the cache to pick up changes in the `package.json`s.